### PR TITLE
Add "How to get and use" block as example template for all x-ray items

### PR DESCRIPTION
**Pull request by brucecrevensten**

### CODE REVIEW

**Comment by cstephen** [comment]
I've added a "get & use this data" section for all remaining items except for:

- CMIP6 indicator items since API documentation, GeoNetwork link, and academic reference(s) are not available for these yet
- Jupyter notebooks since a "get & use this data" section doesn't apply

I've set it up so that there's a root `GetAndUseData.vue` component (with slot) used by all ARDAC data items, and in cases where a family of ARDAC data items share the same info (API link, GeoNetwork link, academic references), I've created an intermediate component that stores all common information.

In other words, for example, we have a `GetAndUseDataHydrology.vue` component that is used by all hydrology ARDAC data items since they all share the same API/GeoNetwork/academic reference info. `GetAndUseDataHydrology.vue` still uses the root `GetAndUseData.vue` component under the hood, but avoids a lot of text duplication between hydrology items.

The `presentInNcr` and `presentInEds` props control whether to display links to NCR or Arctic-EDS, respectively.

To test, look at each of the ARDAC data items and make sure the "get & use this data" content is accurate.
